### PR TITLE
[mkt] minor renames and upgrades for tests

### PR DESCRIPTION
**Pull request by tworec**

### CODE REVIEW

File: core/market/decentralized/tests/test_cyclic_broadcasts.rs [hunk 1]
```diff
@@ -107,9 +107,6 @@ async fn test_unsubscribes_cyclic_broadcasts() -> Result<(), anyhow::Error> {
 
     let market3 = network.get_market("Node-3");
 
-    // Create some offers before we instantiate 3rd node.
-    // This way this 3rd node won't get them in first broadcasts, that
-    // are sent immediately, after subscription is made.
```
Comment (nieznanysprawiciel) [inline core/market/decentralized/tests/test_cyclic_broadcasts.rs:112]: Why is this removed?

File: core/market/decentralized/tests/test_cyclic_broadcasts.rs [hunk 2]
```diff
@@ -310,9 +321,6 @@ async fn test_sharing_someones_else_unsubscribes() -> Result<(), anyhow::Error>
...
```
Comment (nieznanysprawiciel) [inline core/market/decentralized/tests/test_cyclic_broadcasts.rs:315]: Why is this removed?

**Review by nieznanysprawiciel — APPROVED** [review]

File: core/market/decentralized/tests/test_cyclic_broadcasts.rs (hunk 2 above)
Comment (tworec) [reply core/market/decentralized/tests/test_cyclic_broadcasts.rs:315]: because this is not true in this test. Node-3 was created already in line 302

File: core/market/decentralized/tests/test_cyclic_broadcasts.rs (hunk 1 above)
Comment (tworec) [reply core/market/decentralized/tests/test_cyclic_broadcasts.rs:112]: because this is not true in this test. Node-3 was created already in line 99
